src/geometries/Cube.js

```diff
@@ -488,47 +499,6 @@ CubeLevel.prototype.tileHeight = function() {
 };
 
 
-CubeLevel.prototype._validateWithParentLevel = function(parentLevel) {
```

Comment:
We still want to validate that each tile has at most one parent tile in the previous level (i.e., the tile width must be a submultiple of the parent width, and the same for height). Otherwise, `CubeTile#parent` and `CubeTile#children` no longer work correctly.

If you're up for it, you can lift this restriction afterwards (but please make it a separate PR, to keep each change small).